notebooks/itslive/itslive.ipynb

```diff
@@ -0,0 +1,234 @@
+   "outputs": [],
+   "source": [
+    "#1: Now let's render our UI and pick up an hemisphere, if you update the hemisphere you need to execute the cell again.\n",
+    "import warnings\n",
+    "warnings.filterwarnings('ignore')\n",
+    "from itslive import itslive_ui\n",
+    "ui = itslive_ui('global')\n",
+    "ui.render()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#2: We build the parameters to query the ITS_LIVE Search API, we get the time coverage for our selected area\n",
+    "params = ui.build_params()\n",
+    "print(f'current parameters: {params}')\n",
+    "timeline = None\n",
+    "if params is not None:\n",
+    "    timeline = ui.update_coverages()\n",
+    "    total =  sum(item['count'] for item in timeline)\n",
+    "    print(f'Total data granules: {total:,}')\n",
+    "timeline"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#3: Now we are going to get the velocity pair urls, this does not download the files yet just their location\n",
+    "urls = []\n",
+    "params = ui.build_params()\n",
+    "if params is not None:\n",
+    "    urls = ui.get_granule_urls(params)\n",
+    "    # Print the first 10 granule URLs\n",
+    "    for url in urls[0:10]:\n",
+    "        print(url)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Filtering granules by year and month"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#4: This will query AWS(where the granules are stored) so we know the total size of our first N granules\n",
+    "# This may take some time, try reducing the selected area or constraining the other parameters to download a reasonable number of granules.\n",
+    "url_list = [url['url'] for url in urls]\n",
+    "# urls\n",
+    "filtered_urls = ui.filter_urls(url_list, max_files_per_year=5, months=[3,4,5,6,7])\n",
+    "\n",
+    "# max_granules = 100\n",
+    "# sizes = ui.calculate_file_sizes(filtered_urls, max_granules)\n",
+    "# total_zise = round(sum(sizes)/1024,2)\n",
+    "# print(f'Approx size to download for the first {max_granules:,} granules: {total_zise} MB')\n",
+    "print(len(filtered_urls))\n",
+    "filtered_urls"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Downloading the data\n",
+    "**Now that we have our list of data granules we can download them from AWS.**\n",
+    "\n",
+    "If this notebook is running inside AWS we could load the granules into a Dask cluster and reduce our processing times and costs.\n",
+    "Let's get some coffee, some data requests are in the Gigabytes realm and may take a little while to be processed. \n",
+    "Once that your status URL says is completed we can grab the HDF5 data file using the URL on the same response!"
```

Comment:
I never saw a status outputted here but everything completed.

Reply:
I just changed the text